crates/core/migrations/2022-05-19-200746_create_instruction_tables/down.sql

```diff
@@ -0,0 +1,7 @@
+drop table buy_instructions;
+drop table sell_instructions;
+drop table execute_sale_instructions;
+drop table cancel_instructions;
+drop table deposit_instructions;
+drop table withdraw_instructions;
```

Comment:
In an immediate follow up we should index the withdraw_from_treasury instruction. That will allow us to have a print out of each withdraw from the ah treasury, how much, and to what wallet.

@damianr @dandlezzz this is big for marketplace financials reporting.